src/Perl6/Compiler.nqp

```diff
@@ -166,9 +173,23 @@ class Perl6::Compiler is HLL::Compiler {
             $rakudo := "Rakudo™";
         }
 
+        # Support different flavors of Rakudo, i.e. Star.
+        # Rakudo flavor can be either set by an environment variable 'RAKUDO_FLAVOR'
+        # or by the special file '$RAKUDO_PREFIX/share/perl6/site/FLAVOR'
```

Comment:
The repository paths are not an appropriate location for such a file

Reply:
@ugexe @lizmat

Well... It has to be somewhere in $RAKUDO_PREFIX. I'm open for any suggestion. Feel free to make a qualified / better one... and make sure you are aware off
- https://github.com/rakudo/star/pull/198#issuecomment-2054111595
- https://github.com/rakudo/star/pull/198#issuecomment-2056327216

Thanks & regards

Reply:
I dunno where a better place is, but the repository paths are coupled to `CompUnit::Repository::Installation` objects and shouldn't contain files unrelated to them. The `version` file mentioned in one of those links is directly coupled to the `CompUnit::Repository::Installation` that path represents, it isn't a general rakudo version.

Reply:
@ugexe @lizmat

And now? How to continue? Who can clarify and share with me where, within $RAKUDO_PREFIX, to put such a file?

Reply:
What other options (such as creating a new directory) have you considered? I don't exactly have time at the moment to do that type of investigative work, but it only takes me a moment for me to comment on potential options that have already had some thought put into them.

Reply:
@ugexe @lizmat

Well.... `.config` would be a very prominent example, in Windows and UX environments. Look into your $USERHOME, I'd bet you have it, too ;)
The other option could be `.local`.

If a new directory is really an option, I'd vote for `$RAKUDO_PREFIX/.config/FLAVOR`.

Reply:
That seems fine. It looks like the convention used in `$RAKUDO_PREFIX` currently is UNIX-like e.g. `lib`, `bin`, so `etc` might be a good alternative as well. I don't have strong feelings either way.

Reply:
@ugexe @lizmat

1.
Which other existing directories and files in `$RAKUDO_PREFIX` are auto-created / managed by other tools, i.e. by `CompUnit::Repository::Installation`?

2.
So, adding new dir's and files outside of what exists today in `$RAKUDO_PREFIX`, is "ok" due to your knowledge, right?

3.
Can one of you clarify, like in the "steering committee" or whatever the right gremium is, and share how to continue?!

Thanks

Reply:
Invoking the wisdom of @niner in these matters: I simply do not know enough to these intricacies yet  :-(

Reply:
> That seems fine. It looks like the convention used in `$RAKUDO_PREFIX` currently is UNIX-like e.g. `lib`, `bin`, so `etc` might be a good alternative as well. I don't have strong feelings either way.

oki, so changed the lookup path for the new file now to $RAKUDO_PREFIX/etc/FLAVOR`...
hope, this change is woth you approval now....

Thanks for your feedback!